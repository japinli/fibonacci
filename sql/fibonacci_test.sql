CREATE EXTENSION fibonacci;
SELECT fibonacci(0);
SELECT fibonacci(1);
SELECT fibonacci(2);
SELECT fibonacci(3);
SELECT fibonacci(4);
SELECT fibonacci(5);
SELECT fibonacci(6);
SELECT fibonacci(7);
SELECT fibonacci(8);
SELECT fibonacci(9);
SELECT fibonacci(10);
SELECT fibonacci(11);
SELECT fibonacci(12);
SELECT fibonacci(13);
SELECT fibonacci(14);
SELECT fibonacci(15);
SELECT fibonacci(16);
SELECT fibonacci(17);
SELECT fibonacci(18);
SELECT fibonacci(19);
SELECT fibonacci(20);
