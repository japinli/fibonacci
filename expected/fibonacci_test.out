CREATE EXTENSION fibonacci;
SELECT fibonacci(0);
 fibonacci 
-----------
         0
(1 row)

SELECT fibonacci(1);
 fibonacci 
-----------
         1
(1 row)

SELECT fibonacci(2);
 fibonacci 
-----------
         1
(1 row)

SELECT fibonacci(3);
 fibonacci 
-----------
         2
(1 row)

SELECT fibonacci(4);
 fibonacci 
-----------
         3
(1 row)

SELECT fibonacci(5);
 fibonacci 
-----------
         5
(1 row)

SELECT fibonacci(6);
 fibonacci 
-----------
         8
(1 row)

SELECT fibonacci(7);
 fibonacci 
-----------
        13
(1 row)

SELECT fibonacci(8);
 fibonacci 
-----------
        21
(1 row)

SELECT fibonacci(9);
 fibonacci 
-----------
        34
(1 row)

SELECT fibonacci(10);
 fibonacci 
-----------
        55
(1 row)

SELECT fibonacci(11);
 fibonacci 
-----------
        89
(1 row)

SELECT fibonacci(12);
 fibonacci 
-----------
       144
(1 row)

SELECT fibonacci(13);
 fibonacci 
-----------
       233
(1 row)

SELECT fibonacci(14);
 fibonacci 
-----------
       377
(1 row)

SELECT fibonacci(15);
 fibonacci 
-----------
       610
(1 row)

SELECT fibonacci(16);
 fibonacci 
-----------
       987
(1 row)

SELECT fibonacci(17);
 fibonacci 
-----------
      1597
(1 row)

SELECT fibonacci(18);
 fibonacci 
-----------
      2584
(1 row)

SELECT fibonacci(19);
 fibonacci 
-----------
      4181
(1 row)

SELECT fibonacci(20);
 fibonacci 
-----------
      6765
(1 row)

